.github/pull_request_template.md

```diff
@@ -1,5 +1,5 @@
 <!--
-  This template is not mandatory.
+  This template is not optional.
```

Comment:
why ?

Reply:
because I also forgot to remove the `not` word? x_x

Reply:
I thought it was on intend :smiling_imp: